Userland/DevTools/UserspaceEmulator/SoftCPU.cpp

```diff
@@ -1834,7 +1834,18 @@ void SoftCPU::FNDISI(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNCLEX(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNINIT(const X86::Instruction&) { TODO_INSN(); }
 void SoftCPU::FNSETPM(const X86::Instruction&) { TODO_INSN(); }
-void SoftCPU::FLD_RM80(const X86::Instruction&) { TODO_INSN(); }
+
+void SoftCPU::FLD_RM80(const X86::Instruction& insn)
+{
+    VERIFY(!insn.modrm().is_register());
+
+    // long doubles can be up to 128 bits wide in memmory for reasons (alignment) and only uses 80 bits of precision
+    // bit_cast seems to see the 12 bytes, but godbolt says 16 bytes in x86-64 or 12 with -m32
```

Comment:
godbolt is a website. Are you saying that a compiler is telling you something? Please be more clear.

Reply:
Changed it to refer to gcc's behavior